### ISSUE TITLE
Check wrong parameter order

### ISSUE THREAD

**Issue by Nagriar** [algolia/algoliasearch-client-js#165]
it could be cool to check if the size of the API key is greater than the size of the APPID in the constructor to be avoid issue.

**Comment by vvo** [comment]
Yes for this, is it completely safe and future prood? At least for now I guess :)

**Comment by Nagriar** [comment]
Yes it's future proof, the APIKEY will be always greater than the APPID

**Comment by vvo** [comment]
This is already what we do in instantsearch() or docsearch()
